extensions/wikia/PortableInfobox/services/Parser/Nodes/Node.php

```diff
@@ -48,6 +48,19 @@ public function getSourceLabel() {
 		return $sourceLabels;
 	}
 
+	// TODO what happens when there are multiple sources?
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this "TODO" comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)